geonode/base/fixtures/group_test_data.json

```diff
@@ -11,7 +11,7 @@
       "pk": 3,
       "model": "auth.group",
       "fields": {
-        "name": "registered-members",
+        "name": "registered-members1",
```

Comment:
Uhm, I don't like very much this approach. I would spend sometime trying to understand why we get a duplicate. Eventually we might drop the test db instead of reusing it if really needed.

Reply:
@afabiani This key is still failing could we recreate the db. https://app.circleci.com/pipelines/github/GeoNode/geonode/3914/workflows/c71304f3-6c98-48c4-9166-02bd0403fcc9/jobs/12166